xarray/tests/test_backends.py

```diff
@@ -40,6 +41,12 @@
 except ImportError:
     pass
 
+with suppress(ImportError):
+    try:
+        from pathlib import Path
+    except ImportError:
+        from pathlib2 import Path
+
```

Comment:
let's move this to `tests/__init__.py`. We'll also want a `requires_pathlib` defined there.

Reply:
I added the `requires_pathlib` decorator but couldn't remove this nested import block from `test_backends.py`, because `Path()` is explicitly used to set up the pathlib tests.